config/cloudinary.yml

```diff
@@ -12,6 +12,6 @@ default:
   transformation: 't_ddb_cover'
   extension: 'jpg'
 
-auto:
-  transformation: 't_ddb_cover,q_auto'
-  extension: 'jpg'
+#auto:
+#  transformation: 't_ddb_cover,q_auto'
+#  extension: 'jpg'
```

Comment:
Why? And is there a better way? I assume this has to be undone for prod?

Reply:
It has been removed on the current server and I don't remember why, but the links that it creates in the request gives an 404.

Reply:
I think that we concluded that the q_auto was not doing as good a job as expected. Can remember that i was Rolf that can with the auto-quality idea.

Reply:
Removed it.